CSS/CSS.sublime-syntax

```diff
@@ -7,6 +7,14 @@ file_extensions:
   - css.erb
 scope: source.css
 variables:
+  # Many variable names taken directly from https://www.w3.org/TR/CSS22/grammar.html#scanner
+  h: '[0-9a-fA-F]'
+  unicode: '\\{{h}}{1,6}(?:\r\n|[ \t\r\n\f])?'
+  escape: '(?:{{unicode}}|\\[^\r\n\f0-9a-fA-F])'
+  nmstart: '(?:[_a-zA-Z]|[^[:ascii:]]|{{escape}})'
+  nmchar: '(?:[_a-zA-Z0-9-]|[^[:ascii:]]|{{escape}})'
+  ident: '-?{{nmstart}}{{nmchar}}*'
+
```

Comment:
Instead of defining `h`, use the `\h` character class escape. It matches `0-9`, `a-f` and `A-F`. Sublime Text normalizes the line endings in buffers to `\n`, so `\r\n` is not necessary. Thus, `unicode` can be `\\\h{1,6}[ \t\n\f]?` and `escape` can use `{{unicode}}|\\[^\n\f\h]`

The regex engines in Sublime Text are unicode aware, thus `nmstart` should probably be `[[:alpha:]_]|{{escape}}`. That way only unicode alphabetic characters will be matched as the beginning of an identifier, and not things like zero-width spaces. `nmchar` should be `[[:alnum:]_-]|{{escape}}`.